### PR TITLE
feat(i18n) workbench-views-source

### PR DESCRIPTION
**Pull request by Jose-Matsuda**
Implement i18n for classes under workbenc.views.source. This does not include those under views.source.editors.text as those were handled by another PR

### Intent

This PR tackles hard-coded strings to i18n-ify in org.rstudio.studio.client.workbench.views.source (excluding false-positives).
Issue: https://github.com/StatCan/daaas/issues/727

### Approach

This pull-request and future ones will tackle i18nify classes at package-level (e.g. org.rstudio.core) instead of the per-component approach (like we did for all-options-menu)

Implementation follows the same pattern from #9485 as described in gwt/tools/i18n-helpers/README.md. Running the create_dev_locale.sh will create all the new "dev" locale from any new "en" properties files for easier visual confirmation. Refer to README for more information.

### Automated Tests

> Indicate whether this change includes unit tests or integration tests, or link a work item or pull request to add those tests to another repo. If the change cannot or will not be covered by a test, indicate why.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 

Modify an entry in one of the new _en.properties files, and refresh RStudio with ant in devmode.
Run create_dev_locale.sh to clone all _en properties files, which will add a prefix of "@" to each constant and message. Then open RStudio using http://localhost:8787/?locale=dev and browse to the Options->* menus. You can also view in desktop mode by swapping the locale element value attribute from value="en" to value="dev" in RStudio.gwt.xml.

### Checklist

- [ ] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [x] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [ ] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [ ] This PR passes all local unit tests
- [x] Signed the contributor agreement

cc @gtritchie 

## Please NOTE 
There is a file `ChunkOptionsPopupPanel.java` at `/workbench/views/source/editors/text/rmd/display/ChunkOptionsPopupPanel.java` that I was unsure about adding i18n to so I have marked it with a `TODO` as it seems to be used (in a hardcoded fashion) later in the class / in `SetupChunkOptionsPopupPanel.java`. Just want to know if I need to replace all of those with a constant.

### CODE REVIEW

File: src/gwt/src/org/rstudio/studio/client/workbench/views/source/Source.java [hunk 1]
```diff
@@ -14,10 +14,7 @@
  */
 package org.rstudio.studio.client.workbench.views.source;
 
-import com.google.gwt.core.client.JavaScriptObject;
-import com.google.gwt.core.client.JsArray;
-import com.google.gwt.core.client.JsArrayString;
-import com.google.gwt.core.client.Scheduler;
+import com.google.gwt.core.client.*;
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/source/Source.java:17]: Switch back to individual imports, not wildcard.

File: src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourceColumnManager.java [hunk 2]
```diff
@@ -108,6 +108,7 @@
                                             DocumentCloseEvent.Handler,
                                             DebugModeChangedEvent.Handler
 {
+    private static final ViewsSourceConstants constants_ = GWT.create(ViewsSourceConstants.class);
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourceColumnManager.java:111]: nitpick: indentation doesn't match

File: src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourceWindow.java [hunk 3]
```diff
@@ -18,6 +18,7 @@
 import java.util.HashSet;
 import java.util.Set;
 
+import com.google.gwt.core.client.*;
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourceWindow.java:21]: Remove wildcard import, put back individual ones

File: src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourceWindowManager.java [hunk 4]
```diff
@@ -14,10 +14,7 @@
  */
 package org.rstudio.studio.client.workbench.views.source;
 
-import com.google.gwt.core.client.JavaScriptObject;
-import com.google.gwt.core.client.JsArray;
-import com.google.gwt.core.client.JsArrayString;
-import com.google.gwt.core.client.Scheduler;
+import com.google.gwt.core.client.*;
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/source/SourceWindowManager.java:17]: revert wildcard import

File: src/gwt/src/org/rstudio/studio/client/workbench/views/source/ViewsSourceConstants.java [hunk 5]
```diff
@@ -0,0 +1,2725 @@
+package org.rstudio.studio.client.workbench.views.source;
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/source/ViewsSourceConstants.java:1]: Move package line below comment block.

File: src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/display/ChunkOptionsPopupPanel.java [hunk 6]
```diff
@@ -202,36 +203,38 @@ else if (value == OUTPUT_SKIP_THIS_CHUNK)
...
       panel_.add(nameAndOutputGrid);
       
       showWarningsInOutputCb_ = makeTriStateToggle(
-            "Show warnings",
+            constants_.showWarnings(),
             "warning");
       panel_.add(showWarningsInOutputCb_);
-      
+
+      // TODO these 'option's seem to be used later(marked with TODO), do these need translations?
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/rmd/display/ChunkOptionsPopupPanel.java:218]: Yes, these are used as you described, and should not be localized. Please remove the TODO comments.

**Review by gtritchie — CHANGES_REQUESTED** [review]

**Comment by Jose-Matsuda** [comment]
Will address merge conflicts shortly and will push some commits. Will squash into two commits (1st is the initial and 2nd will be the review + conflict merge) and will re-request review when ready

**Review by gtritchie — APPROVED** [review]

**Pull request merged**